book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
```

Comment:
- [ ] I suggest that `@rentals` should be initialized here as an empty array to store rentals. For more info about constructor and instance vars please visit :point_right: [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars)